docs/src/components/Example.tsx

```diff
@@ -1,6 +1,8 @@
-export function Example({ children }) {
+export function Example({ children, className = '' }) {
```

Comment:
Adding the ability to add a className to an Example so we can add scoped CSS for examples where we show overriding CSS for primitives without adding extra classNames or IDs to the primitive. For example, the code we show on the docs:

```css
.amplify-tabs {
  background-color: var(--amplify-colors-background-secondary);
}

.amplify-tabs-item {
  background-color: var(--amplify-colors-background-primary);
}
```

Currently we mimic that behavior by either adding classNames or style props on the primitives in the example like [here](https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/ui/primitives/alert/react.mdx):

```css
/* styles.css */
:root {
  --amplify-components-alert-background-color: yellow;
}
/* OR */
.amplify-alert {
  background-color: yellow;
}
```

```jsx
<Alert backgroundColor="yellow">Change the default Alert to yellow</Alert>
```

Being able to add a className to an example block will allow us to write example CSS that more closely resembles the actual CSS showed in the code block, like: 

Example code in docs:

```css
/* styles.css */
/* When a Tab is selected and displaying its corresponding content panel */
.amplify-tabs-item[data-state='active'] {
  background-color: var(--amplify-colors-brand-primary-20);
}

/* When a Tab is disabled */
.amplify-tabs-item[data-disabled] {
  background-color: var(--amplify-colors-background-tertiary);
}
```

Code in CSS

```css
.tabs-data-demo .amplify-tabs-item[data-state='active'] {
  background-color: var(--amplify-colors-brand-primary-20);
}

.tabs-data-demo .amplify-tabs-item[data-disabled] {
  background-color: var(--amplify-colors-background-tertiary);
}
```